android/apollo/src/com/andrew/apollo/loaders/RecentSongLoader.java

```diff
@@ -0,0 +1,54 @@
+package com.andrew.apollo.loaders;
```

Comment:
please add license header

Reply:
Keep this in your notes.
Whenever you touch an existing file, the header should be updated to include your name and the year should be changed from 2016 or 2017 to 2018

```
/*
 * Created by Angel Leon (@gubatron), Alden Torres (aldenml)
 * Marcelina Knitter (@marcelinkaaa), José Molina (@votaguz)
 * Copyright (c) 2011-2018 FrostWire(R). All rights reserved.
 
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```